### ISSUE TITLE
New / old order appears to be inconsistent in log "Detailed information about changes"

### ISSUE THREAD

**Issue by bobOnGitHub** [aide/aide#128]
In the AIDE output below it _appears_ that the current state of `/root/.ssh/known_hosts` and   `/etc/ld.so.cache` are on the right wheras the current state of `/etc/resolv.conf` is on the left.

From aide log 

```
---------------------------------------------------
Detailed information about changes :
---------------------------------------------------

File: /etc/ld.so.cache
  SELinux  : unconfined_u:object_r:ld_so_cach | system_u:object_r:ld_so_cache_t:
             e_t:s0                           | s0

File: /etc/resolv.conf
  Size     : 76                               | 30
  SHA512   : vWpV3eoqCF3+WDsEBwLAF9mwzAjEdgMA | j3W8R6kCOmHS0WVeARtdyC9tGHCPDupx
             WgsEmBqSF1wuxERaJmRkgFWuL8MpbQf1 | KMkB2/gLbRHtd+fiW5FqqYYyqnucrA0h
             DjribV7jI1ldnn43WaTysA==         | rluxYUfx+cQ9J9bfLz0V4A==

File: /root/.ssh/known_hosts
  Perm     : -rw-r--r--                       | -rw-------
  ACL      : A: user::rw-                     | A: user::rw-
             A: group::r--                    | A: group::---
             A: other::r--                    | A: other::---
```

From console
```
[root@localhost ~]# stat /etc/ld.so.cache
  File: /etc/ld.so.cache
  Size: 94707           Blocks: 192        IO Block: 4096   regular file
Device: fd01h/64769d    Inode: 203901633   Links: 1
Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
Context: system_u:object_r:ld_so_cache_t:s0
Access: 2022-03-30 19:47:14.207695728 +0100
Modify: 2022-03-30 19:47:14.076695729 +0100
Change: 2022-03-30 19:47:14.205695728 +0100
 Birth: -

[root@localhost ~]# stat /etc/resolv.conf
  File: /etc/resolv.conf
  Size: 76              Blocks: 8          IO Block: 4096   regular file
Device: fd01h/64769d    Inode: 204136029   Links: 1
Access: (0644/-rw-r--r--)  Uid: (    0/    root)   Gid: (    0/    root)
Context: system_u:object_r:net_conf_t:s0
Access: 2022-03-31 12:21:41.353055346 +0100
Modify: 2022-03-31 12:21:40.429060059 +0100
Change: 2022-03-31 12:21:40.467059866 +0100
 Birth: -

[root@localhost ~]# stat  /root/.ssh/known_hosts
  File: /root/.ssh/known_hosts
  Size: 569             Blocks: 8          IO Block: 4096   regular file
Device: fd01h/64769d    Inode: 89856015    Links: 1
Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
Context: unconfined_u:object_r:ssh_home_t:s0
Access: 2022-03-30 05:15:22.163801794 +0100
Modify: 2022-03-29 22:07:20.723595005 +0100
Change: 2022-03-29 22:07:20.723595005 +0100
 Birth: -
[root@localhost ~]# 
```
```
[root@localhost ~]# aide --version
Aide 0.16

Compiled with the following options:

WITH_MMAP
WITH_PCRE
WITH_POSIX_ACL
WITH_SELINUX
WITH_XATTR
WITH_E2FSATTRS
WITH_LSTAT64
WITH_READDIR64
WITH_ZLIB
WITH_CURL
WITH_GCRYPT
WITH_AUDIT
CONFIG_FILE = "/etc/aide.conf"
```

Operating System: Rocky Linux 8.4
KDE Plasma Version: 5.18.4
KDE Frameworks Version: 5.68.0
Qt Version: 5.12.5
Kernel Version: 4.18.0-305.25.1.el8_4.x86_64
OS Type: 64-bit

**Comment by hvhaugwitz** [comment]
AIDE v0.16 has been released several years ago, please try to reproduce your issue with the latest upstream version (`0.17.4`).

**Comment by bobOnGitHub** [comment]
Hi @hvhaugwitz,

I installed via the Rocky Linux 8.4 repo. (dnf install aide)
The current version installing from the latest RL 8.5 is the same:
```
[bob@localhost ~]$ aide --version
bash: aide: command not found...
Install package 'aide' to provide command 'aide'? [N/y] y


 * Waiting in queue... 
The following packages have to be installed:
 aide-0.16-14.el8_5.1.x86_64    Intrusion detection environment
Proceed with changes? [N/y] 

```

So this is what the latest RL and hence the latest RHEL are using.... 
I've found refs to v0.15 being in RHEL 7 so it looks like they are updating but just lagging a bit - should get v.017 in RL 9 if the pattern holds.

(Current) Fedora 35 (Workstation Edition)  is also using AIDE v 0.16

I stick with official repo software where I can because 
1) I tell myself the distro people have checked it / I don't have to
2) Non-repo versions might not work with the distro / have compatability issues I won't know about 'til they happen
3) Trying to build sw is rarely painless and usually involves a lot more time than initially suggested

So....given its only a possible non-critical bug that may no longer be present and I'm not going to move to verion .017 ahead of RHEL I'll leave it here.

I do apologise for not getting into it (build latest/try to reproduce etc) but I have a massive and ever-growing todo list as it is.

I have to wonder though how much of your user-base is on v.016 and if maintenance of / patches to that wouldn't make sense ? 

In any case cheers for the quick reply and thanks for the sw :+1:  (12 yrs is some commitment !)
